health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/utils/setupCampaignValidators.js

```
@@ -292,16 +292,48 @@ export const  handleValidate = ({formData,t,setShowToast,hierarchyDefinition,low
+    //   const cycleData = formData?.cycleConfigure?.cycleData || [];
+
+    //   if (cycleNumber === "" || cycleNumber === 0 || deliveryNumber === "" || deliveryNumber === 0) {
+    //     setShowToast({ key: "error", label: "DELIVERY_CYCLE_ERROR" });
+    //     return false;
+    //   } else {
+    //     setShowToast(null);
+    //     return true;
+    //   }
+
     case "cycleConfigure":
       const cycleNumber = formData?.cycleConfigure?.cycleConfgureDate?.cycle;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)